res/database/Default/user/therob.xml

```diff
@@ -2412,18 +2725,22 @@
 			<title>
 				<de>%TITLE%</de>
 				<en>%TITLE%</en>
+				<pl>%TITLE%</pl>
 			</title>
 			<description>
 				<de>Familienkomödie über das Leben mit einem frechen Sohnemann. Das Drehbuch wird von dem Stardrehbuchschreiber Don Ron geschrieben.</de>
 				<en>Family comedy about life with a mischievous son. The script is written by star screenwriter Don Ron.</en>
+				<pl>Rodzinna komedia o życiu z psotnym synem. Scenariusz został napisany przez scenarzystę Dona Rona.</pl>
 			</description>
 			<variables>
 				<title>
 					<de>Engel oder Bengel|%NAME% dreht durch|%NAME% außer Rand und Band</de>
 					<en>Angel or Rascal|%NAME% goes berserk|%NAME% goes haywire</en>
+					<pl>Anioł lub łobuz|%NAME% - diabełek w owczej skórze|%NAME% rozrabia</pl>
 				</title>
 				<name>
 					<de>%PERSONGENERATOR_FIRSTNAME(de,1)%|%PERSONGENERATOR_FIRSTNAME(es,1)%|Tim</de>
+					<pl>Pawełek|%PERSONGENERATOR_FIRSTNAME(es,1)%|Marcelek</pl>
```

Comment:
use pl-person generator for first alternative

Reply:
changed to <pl>Pawełek|%PERSONGENERATOR_FIRSTNAME(pl,1)%|Marcelek</pl>

Reply:
Pawełek and Marcelek is diminutive and fits the part. I suggest leaving it that way.